source/exceptions.tex

```diff
@@ -1108,8 +1109,6 @@
 
 \pnum
 \indextext{\idxcode{terminate}}%
```

Comment:
Is it useful to add another index entry here?

Reply:
Taking a quick look at the indexing of `std::terminate` in general, I would like to defer that question as it is not a change introduced by this PR.  We are removing only the first reference to `std::terminate` in this paragraph, not all as another two remain.

If we look at the indexing of `std::terminate`, we might want to do a more focussed pass, including the library indexing as well.

Reply:
Agreed, let's fix the indexing later.